pom.xml

```diff
@@ -1019,6 +1020,13 @@
         <version>1.2.0</version>
       </dependency>
 
+      <!-- Upgrade azure-data-lake-store-sdk dependency-->
+      <dependency>
+        <groupId>org.wildfly.openssl</groupId>
+        <artifactId>wildfly-openssl</artifactId>
+        <version>${wildfly-openssl.version}</version>
+      </dependency>
+
```

Comment:
Thanks for this! Do we also need an entry for `org.wildfly.openssl:wildfly-openssl-java`? I see it was there in the original version.

Reply:
it's part of `org.wildfly.openssl:wildfly-openssl` you no need.

Reply:
Also the usage of these libs are coming from ADLS PinotFS, which I think is already deprecated. 
Can your team also check and verify the usage and maybe just remove it? https://github.com/apache/pinot/issues/12598